websites/C/Cisco Netwoking Academy/dist/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Cisco Networking Academy",
+    "author": {
+      "name": "YJJcoolcool",
+      "id": "300929818231308288"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/pvz4KSf.png",
```

Comment:
Better logo?

Reply:
I replaced it with another logo, but it's a rectangle, so I'm not sure if it's suitable or not. If not, I'll put back the old logo.

Reply:
Only square logo.

Reply:
Ok, then I'll use the old logo. I couldn't find any high-quality square ones so this is the best one I could find.